.github/pull_request_template.md

```diff
@@ -34,8 +43,7 @@ Screenshots or video that presents the feature or fix
 All those checkboxes should be marked before submitting the PR
 -->
 
-- [ ] New functionality is covered by Analytics events
-- [ ] Analytics events are documented
-- [ ] Analytics dashboard is updated
 - [ ] Functionality covered by unit tests
-- [ ] I have manually tested if the code works
+- [ ] Functionality covered by integration tests
+- [ ] I have updated PR description with relevant information
+- [ ] I have manually tested if the code and app works
```

Comment:
Suggestion: We can add `I've done self code review`. It's really helpful for other reviewers ;)